### PR TITLE
feat: Update PyPI souce

### PR DESCRIPTION
**Pull request by tachikoma-li**
This is recommended by DevOps for security concern and discussed/approved in [CoP](https://docs.google.com/document/d/1tev0WmJDhOyIqA40gr5thEeAUmidyLaIc0Pu0oORCmM/edit#heading=h.wok2hrmo1zhz).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -108,8 +108,13 @@ Twitter = "https://twitter.com/qctrlHQ"
...
+# Use the internal PyPI as the main source: DO NOT EDIT
+[[tool.poetry.source]]
+name = "Q-CTRL PyPI"
+url = "https://pypi.q-ctrl.com/simple"
+priority = "primary"
+
+# Use the public PyPI as the supplemental source: DO NOT EDIT
+[[tool.poetry.source]]
+name = "PyPI"
+priority = "supplemental"
```
Comment (kdav108) [inline pyproject.toml:120]: Hey @tachikoma-li, perhaps we should mention that for repos that don't require internal packages, public PyPI can still be used as the primary. Thoughts?

**Comment by kdav108** [comment]
Also I updated the PR description with a link to CoP discussion on this, just FYI.

File: pyproject.toml (hunk 1 above)
Comment (tachikoma-li) [reply pyproject.toml:120]: Good point. Maybe the whole section should remain as a comment as before?

Comment (kdav108) [reply pyproject.toml:120]: Yep I think that works!

File: pyproject.toml [hunk 2]
```diff
@@ -110,7 +110,13 @@ YouTube = "https://www.youtube.com/qctrl"
 GitHub = "https://github.com/qctrl"
 
 # Uncomment the block below if the package requires access to the Internal PyPI.
```
Comment (kdav108) [inline pyproject.toml:112]:
```suggestion
# Uncomment the following 2 blocks if the package requires access to the Internal PyPI.
```
Just to make it very clear that you also need to uncomment the PyPI as 'supplemental' configuration block.

**Review by kdav108 — APPROVED** [review]
Thanks @tachikoma-li!

**Pull request merged**